pagopa/io-app
# fix: [IOBP-915] Error toast message when download receipt is failed due left response

## Short description
This PR shows the toast error message when the download of a receipt from biz-events fails due a `left` response;

## List of changes proposed in this pull request
- Inoked the `onError` event when received a left response from the server in order to show a toast message;

## How to test
- Open a transaction details from the "Payments" section;
- Tap on the `Ottieni ricevuta` CTA;
- Simulate, if possible, a left response from the server with a man-in-the-middle (i.e. Proxyman) and check that the toast message appears;
